scalafmt-core/shared/src/main/scala/org/scalafmt/config/IndentOperator.scala

```diff
@@ -46,7 +46,11 @@ object IndentOperator {
     ScalafmtConfig.indentOperatorsExcludeAkka)
   implicit val IndentOperatorDecoder: ConfDecoder[IndentOperator] =
     ConfDecoder.instance[IndentOperator] {
-      case Conf.Str("spray") => Ok(IndentOperator.akka)
-      case els => default.reader.read(els)
+      case Conf.Str("spray") | Conf.Str("akka") | Conf.Str("akka-http") =>
```

Comment:
`Conf.Str("spray" | "akka" | "akka-http")`

Reply:
Oh wow, I did not know I could do that